src/main/java/ai/zuva/docai/DocAIClient.java

```diff
@@ -72,6 +73,16 @@ private static MediaType toMediaType(String... contentType) {
     return mediaType;
   }
 
+  public static String listToQueryParams(String key, List<String> requestIds) {
```

Comment:
The remaining uses for it are used by wiremock to build the url. I don't believe it makes sense to use `buildUrl` to build those URLs, because at that point it's not really testing anything anymore.

Reply:
Ideally we'd use WireMock's query parameter helpers, but it looks like they don't do what we need: https://stackoverflow.com/questions/40934019/multi-value-querystring-in-wiremock

Do you want to move it to the TestHelpers class if it's just for tests?

Reply:
Yea that makes sense! 706386af6d5a907ad2dee2f93e277ab6f74a0f88